File: myapp/src/app/model/order.ts
export class Order {
    id:number;
    name:String;
    data_create:Date;
    data_update:Date;
    description:String;
    img:String;
    price:Number;

    constructor(id:number,name:String,data_create:Date,data_update:Date,description:String,image:String,price:Number){
        this.id=id;
        this.name=name;
        this.data_create=data_create;
        this.data_update=data_update;
        this.description=description;
        this.img=image;
        this.price=price;
    }


}

File: back-end/src/main/java/com/spring/restaurant/model/Order.java
package com.spring.restaurant.model;

import com.fasterxml.jackson.annotation.JsonIgnore;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name =  "orderfood")
public class Order extends caregory_order {
    @Column
    private int price;
    @Column(name = "image")
    private String img;

    @Column
    @Lob
    private  String description;

    @JsonIgnore
    @ManyToOne
    @JoinColumn(name = "id_category")
    private Category category;
}
